doc/rose-rosie-advice.html

```diff
@@ -0,0 +1,119 @@
+
+    <p><a href=".">Rose Documentation</a></p>
+  </div>
+
+  <div id="body-main">
+    <h1>Rose Micellaneous: rosie create v copy v checkout</h1>
+
+    <div id="content"></div>
+
+    <h2 id="rosie create"><var>rosie create</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Allows the user to generate a new suite and provides a new unique suite id 
+    which is provided by the roses repository. E.g.:</p>
+    <pre class="shell">
+rosie create
+</pre>
+
+    <h2 id="rosie copy"><var>rosie copy</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Copies an existing suite using its <var>$SUITE_ID</var> from the roses 
+    repository to a new suite id which is provided by the roses repository. 
+    E.g.:</p>
+    <pre class="shell">
+rosie copy $SUITE_ID
+</pre>
+
+    <p>The suite is assigned to the user invoking rosie copy, making them the owner unless they 
+    change this in the <var>rose-suite.info</var> file. 
+    The <var>rose-suite.info</var> is opened in an editor for the user to edit 
+    appropriately when the suite is being created. 
+    The <var>rose-suite.info</var> should provide information from the user to 
+    make the suite easily discoverable by themselves and others. 
+    Add information to the <var>title</var> and <var>project</var> as 
+    required to make it meaningful and easily searchable using tools such 
+    as:</p>
+    <pre class="shell">
+    rosie go
+</pre>
+    <p>the suite browser GUI tool provided by rose.
+    or:</p>
+    <pre class="shell">
+    rosie lookup/ls
+</pre>
+    <p>the command line terminal search of all of the users suites in the 
+    roses repository.</p>
+
+    <h2 id="rosie checkout"><var>rosie checkout</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Checks out an existing suite specified by $SUITE_ID from the roses repository to your local roses/ directory using the original $SUITE_ID. E.g.:</p>
+    <pre class="shell">
+rosie checkout $SUITE_ID
+</pre>
+
+    <p>This suite is not a new unique copy and the original owner still owns 
+    the suite and therefore the only person that can commit back to the 
+    trunk of that suite. 
+    If desired, the suite owner can add the person to the access list (in the 
+    <var>rose-suite.info</var> file) which then would give the <var>named</var> 
+    person full access to the suite. 
+    The suite owner may decide not to give trunk access to other 
+    developers/users but can themselves merge others branch changes in to the 
+    trunk of their suite.</p>
+
+    <p>For developers/users that checkout a suite and want to branch from the 
+    suite the users should refer to their organisations version control system.
+    If you are using fcm as a wrapper to svn then:</p>
+    <pre class="shell">
+rosie checkout $SUITE_ID
+cd roses/$SUITE_ID
+fcm bc $BRANCH_NAME
+fcm sw $BRANCH_NAME
+fcm info
+</pre>
+    <p>Meanings of the fcm commands:</p>
+    <pre class="prettyprint lang-rose_conf">
+bc = branch create
```

Comment:
I think rather than defining what the fcm commands are, it'd be better to add comments inline in the above worked example. e.g.
```
fcm bc $BRANCH_NAME    # Create new branch
fcm sw $BRANCH_NAME   # Swap to branch
```